coordinator/seal/seal.go

```diff
@@ -186,7 +186,7 @@ func (s *AESGCMSealer) SetEncryptionKey(encryptionKey []byte) error {
 // backupEncryptionKey creates a backup of an existing seal key
 func (s *AESGCMSealer) backupEncryptionKey() {
 	if sealedKeyData, err := ioutil.ReadFile(s.getFname(SealedKeyFname)); err == nil {
-		t := time.Now()
+		t := time.Now() // TODO(katexochen): trusted time?
```

Comment:
I've added trusted time to the `AESGCMSealer`. However, this won't prevent the named attack, since an malicious host could just block any connection to trusted time servers, and `ttime.Now()` would always return the same default `time.Time{}`.